src/components/Pattern/Pattern.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PatternWrapper } from './Pattern.styles';
+import { smallSVG, mediumSVG, largeSVG } from './Pattern.svgs';
+
+const Pattern = ({ color = 'midblue', size = 'M' }) => {
+  let svg;
+
+  switch (size) {
+    case 'S':
+    case 's':
+      svg = smallSVG;
+      break;
+    case 'M':
+    case 'm':
+      svg = mediumSVG;
+      break;
+    case 'L':
+    case 'l':
+      svg = largeSVG;
+      break;
+    default:
+  }
+
+  return (
+    <PatternWrapper color={color}>
+      <div dangerouslySetInnerHTML={svg} />
```

Comment:
I don't have any experience with this. @RobTF9 is the build warning about using this property an issue?

For my own knowledge, why did you use this approach (importing the path as a string of HTML)? I'm so unfamiliar with SVGs so anything I can glean here would be helpful!